jps-plugin/src/org/intellij/erlang/jps/builder/ErlangBuilder.java

```diff
@@ -209,6 +196,7 @@ private static void runErlc(ErlangTarget target,
                               boolean isTest) throws ProjectBuildException, IOException {
     GeneralCommandLine commandLine = getErlcCommandLine(target, context, compilerOptions, outputDirectory, erlangModulePathsToCompile, isTest);
     Process process;
+    LOG.debug("Run erlc compiler with command" + commandLine.getCommandLineString());
```

Comment:
It will be hard to read this log message - there is no space before command line string